src/libsync/configfile.cpp

```diff
@@ -252,26 +252,18 @@ QVariant ConfigFile::getPolicySetting(const QString &setting, const QVariant &de
 
 QString ConfigFile::configPath() const
 {
-#ifndef TOKEN_AUTH_ONLY
     if (_confDir.isEmpty()) {
```

Comment:
Can you make a comment on https://github.com/owncloud/client/issues/684 and explain what you change for 2.5? Also does the milestone need to be set for that issue?

Reply:
^^ FYI